### ISSUE TITLE
Small suggestions for documentation

### ISSUE THREAD

**Issue by baskren** [RaphaelJenni/FirebaseUI-Angular#1]
First, thank you VERY MUCH for putting this documentation together.  It is greatly appreciated.  As someone new to Angular2 and Firebase, I did get tripped up on something small in your **How to use** documentation.  I wasn't quite able to put all the pieces together (again, I'm new) as to what you meant in the **Add the firebaseui css to your imports:** section.  

It only took me about 10 minutes to realize that the sample code in that section was from the example project's `.angular-cli.json` file.  Yet, I kept getting results like the following:

![image](https://cloud.githubusercontent.com/assets/2528888/25356902/fd006608-2909-11e7-8f9c-1d7c83b8a813.png)

It took me about an hour to realize that I needed to add `@import url("../node_modules/firebaseui/dist/firebaseui.css");` to my project's `src/styles.css` file.   Success!

But that got me thinking.  It seemed that the change to `.angular-cli.json` was redundant.  So I tried returning `.angular-cli.json` back to its original form and, as expected, the buttons still rendered correctly.

Again, thanks for sharing this project and taking the time to document it so well.

***tl dr:*** **Angular-CLI** section of **How to use** documentation was a red herring.  Please replace that section with `@import url("../node_modules/firebaseui/dist/firebaseui.css");` to example project's `src/styles.css` file.

**Comment by RaphaelJenni** [comment]
@baskren Hey, nice that I could help.

Actually adding the css to the .angular-cli.json should be enough... Have you restarted the server (`ng serve`)?

Else I have to check that again.

**Comment by RaphaelJenni** [comment]
It works for me.
I've added a sample: https://github.com/RaphaelJenni/FirebaseUI-Angular/tree/master/sample

You can check that out and say me if it worked 😃

**Comment by baskren** [comment]
I'll give restarting the server a try.

**Comment by baskren** [comment]
Yes, closing (^-C) the server and restarting (`ng server --open`) was required for the suggested change to the `.angular-cli.json` file to work.  So, in other words, changing `.angular-cli.json` is probably more *correct* while adding `@import url("../node_modules/firebaseui/dist/firebaseui.css");` to `src/styles.css` appears to be more fool-proof (that would be me).

**Comment by RaphaelJenni** [comment]
Nice to hear, that it worked 😃
